src/madetomeasure_api/routes/subscribers.clj

```diff
@@ -1,12 +1,39 @@
 (ns madetomeasure-api.routes.subscribers
   (:require [madetomeasure-api.layout :as layout]
-            [compojure.core :refer [defroutes GET POST DELETE PATCH]]))
+            [madetomeasure-api.db.core :as db]
+            [cheshire.core :refer :all]
+            [compojure.core :refer [defroutes GET POST DELETE PATCH]]
+            [madetomeasure-api.middleware.validate-json :refer [json-schema-validate]])
+  (:import java.sql.SQLException))
+
+(def default-subscriber-map
+  ^{:private true}
+  {:first_name nil :last_name nil})
```

Comment:
So something that I find interesting about yesql is that it requires that you have all parameters even if they are null.

For instance if I send

`{:address "flap@gums.com"}` yesql will barf and say hey you didn't add first_name / last_name.

I think this is actually kind of a feature in a way cause it makes you explicitly say that the default-subscriber is `{:first_name nil :last_name nil}` which you then can merge into whatever is pushed in.

Potentially we could leave it as is but it'd require people to fill out first_name=null last_name=null which I think is kind of weird